src/model/room/Room.java

```diff
@@ -0,0 +1,67 @@
+package model.room;
+
+public abstract class Room {
+    protected String status = "Available";
+    protected int noOfRoom;
+    protected int noOfBeds;
```

Comment:
[Fix] avand in vedere ca tu ai deja enumul cu SINGLE, DOUBLE, TRIPLE, care reprezinta cate paturi sunt in camera, nu vad la ce e folosit acest camp.